code/obj/machinery/dialysis.dm

```diff
@@ -0,0 +1,169 @@
+TYPEINFO(/obj/machinery/dialysis)
+	mats = list("MET-1" = 20, "CRY-1" = 5, "CON-2" = 5)
+
+/obj/machinery/dialysis
+	name = "dialysis machine"
```

Comment:
"Space"

Reply:
No.

Reply:
This and the response shouldn't have made me giggle. Heheh.